--- json-c-0.15/arraylist.c	2022-09-28 20:17:09.119269290 +0800
+++ json-c-0.15-new/arraylist.c	2022-10-25 14:26:41.066622270 +0800
@@ -146,9 +146,10 @@
 		if(arr->length <= idx)
 			arr->length = idx + 1;
 	} else {
-		for (size_t i = idx; i < arr->length - 1; i++)
-			arr->array[i] = arr->array[i+1];
+		size_t i;

+		for (i = idx; i < arr->length - 1; i++)
+			arr->array[i] = arr->array[i+1];
 		if (idx < arr->length) {
 			arr->array[arr->length-1] = NULL;
 			arr->length--;
